pkg/render/intrusion_detection.go

```diff
@@ -1453,22 +1146,204 @@ func (c *intrusionDetectionComponent) intrusionDetectionElasticsearchAllowTigera
 	}
 }
 
-func (c *intrusionDetectionComponent) adAPIAllowTigeraPolicy() *v3.NetworkPolicy {
-	return &v3.NetworkPolicy{
-		TypeMeta: metav1.TypeMeta{Kind: "NetworkPolicy", APIVersion: "projectcalico.org/v3"},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      ADAPIPolicyName,
-			Namespace: IntrusionDetectionNamespace,
+// adComponentsToDelete returns a list of objects to delete. Anomaly detection used to be installed here,
+// but has since been removed. This function is kept around to clean up any old objects that may be left.
+func (c intrusionDetectionComponent) adComponentsToDelete() []client.Object {
+	objs := []client.Object{
+		&corev1.ServiceAccount{
+			TypeMeta: metav1.TypeMeta{Kind: "ServiceAccount", APIVersion: "v1"},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      ADAPIObjectName,
+				Namespace: c.cfg.Namespace,
```

Comment:
Since AD is old and deprecated should these namespaces stay hardcoded to the one where they were installed originally? Seems unlikely but also doesn't make sense to me to have a namespace referenced here where they may have never existed anyway.

Reply:
Yep, they should stay hardcoded I think. Strictly speaking the behavior should be the same, but it's more clear if we just leave them hard-coded. This was a find / replace error I think.